pkg/server/ekadashi.go

```diff
@@ -28,5 +32,7 @@ func (s *EkadashiServer) nextEkadashiHandler(w http.ResponseWriter, r *http.Requ
 		jsonError(w, http.StatusInternalServerError, fmt.Errorf("cannot get next ekadashi day: %v", err))
 		return
 	}
-	jsonResponse(w, http.StatusOK, ekadashiDate.Date)
+	jsonResponse(w, http.StatusOK, ekadashiJSON{Date: fmt.Sprintf("%v %s %v", ekadashiDate.Date.Day(),
```

Comment:
Use time.Format

Reply:
Well. I added a lot more neat solution for this task. Cheers!